ValveKeyValue/ValveKeyValue/ObjectCopier.cs

```diff
@@ -123,14 +124,7 @@ static KVValue ConvertObjectToValue(Type objectType, object managedObject, IObje
                         continue;
                     }
 
-                    if (TryConvertValueTo<string>(member.Name, member.Value, out var convertedMemberValue))
```

Comment:
I removed this because calling `CopyObject` would eventually do the cast to string above, any reason not to do this?